python/src/main/python/drivers/graphicsfuzz_piglit_converter.py

```diff
@@ -81,7 +82,7 @@ def make_shader_test_string(shader_job: str) -> str:
     shader_test_string += make_require_header(shader_version_header) + '\n'
     shader_test_string += make_vertex_shader_header() + '\n'
     shader_test_string += make_fragment_shader_header(shader_file_string) + '\n'
-    shader_test_string += make_test_header(shader_job_json_parsed)
+    shader_test_string += make_test_header(shader_job_json_parsed, nodraw)
```

Comment:
I suggest adding the `clear color 0.0 0.0 0.0 1.0` command, even though it probably won't matter, unless shader_runner definitely does that anyway.

Reply:
That's probably a good idea - just for sanity.